docs/content/en/docs/reference/baremetal/bare-custom-hookos.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Customize HookOS for EKS Anywhere on Bare Metal"
+linkTitle: "Customize HookOS for EKS Anywhere on Bare Metal"
```

Comment:
Can we remove all the "for EKS Anywhere on Bare Metal" from the link titles? It makes the side bar harder to read and a bit cluttered/redundant

Reply:
Yes. I agree. Done.